2018-08-05T17:44:20.5608642-04:00 0HLFR3ICVB1RD:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44344/   (ca22a1cb)
2018-08-05T17:44:20.5608979-04:00 0HLFR3ICVB1RF:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:44344/api/AccountApi/DoLogin application/json 0 (ca22a1cb)
2018-08-05T17:44:20.5609512-04:00 0HLFR3ICVB1RE:00000001 [INF] Request starting HTTP/1.1 DEBUG http://localhost:50400/  0 (ca22a1cb)
2018-08-05T17:44:20.6751104-04:00 0HLFR3ICVB1RE:00000001 [INF] Request finished in 146.9759ms 200  (791a596a)
2018-08-05T17:44:20.9662710-04:00 0HLFR3ICVB1RD:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:20.9704846-04:00 0HLFR3ICVB1RF:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.0258559-04:00 0HLFR3ICVB1RD:00000001 [INF] The file "/index.html" was not modified (f1f8d725)
2018-08-05T17:44:21.0781152-04:00 0HLFR3ICVB1RD:00000001 [INF] Request finished in 569.7472ms 304 text/html (791a596a)
2018-08-05T17:44:21.1070335-04:00 0HLFR3ICVB1RE:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44344/polyfills.9f773b75ffcda6bb2a95.js   (ca22a1cb)
2018-08-05T17:44:21.1447059-04:00 0HLFR3ICVB1RD:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44344/main.885a482e6ccfa9258557.js   (ca22a1cb)
2018-08-05T17:44:21.1479952-04:00 0HLFR3ICVB1RG:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44344/styles.48153a45653ae241b04d.css   (ca22a1cb)
2018-08-05T17:44:21.1888382-04:00 0HLFR3ICVB1RE:00000002 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.2296576-04:00 0HLFR3ICVB1RI:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44344/runtime.a66f828dca56eeb90e02.js   (ca22a1cb)
2018-08-05T17:44:21.2420621-04:00 0HLFR3ICVB1RE:00000002 [INF] The file "/polyfills.9f773b75ffcda6bb2a95.js" was not modified (f1f8d725)
2018-08-05T17:44:21.2463851-04:00 0HLFR3ICVB1RG:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.2802821-04:00 0HLFR3ICVB1RD:00000002 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.3002522-04:00 0HLFR3ICVB1RH:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44344/scripts.e037fa6b4f7e71ff8752.js   (ca22a1cb)
2018-08-05T17:44:21.3369998-04:00 0HLFR3ICVB1RI:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.3828598-04:00 0HLFR3ICVB1RG:00000001 [INF] The file "/styles.48153a45653ae241b04d.css" was not modified (f1f8d725)
2018-08-05T17:44:21.4021845-04:00 0HLFR3ICVB1RD:00000002 [INF] The file "/main.885a482e6ccfa9258557.js" was not modified (f1f8d725)
2018-08-05T17:44:21.4165374-04:00 0HLFR3ICVB1RH:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.4503832-04:00 0HLFR3ICVB1RI:00000001 [INF] The file "/runtime.a66f828dca56eeb90e02.js" was not modified (f1f8d725)
2018-08-05T17:44:21.4911246-04:00 0HLFR3ICVB1RE:00000002 [INF] Request finished in 333.3767ms 304 application/javascript (791a596a)
2018-08-05T17:44:21.4949268-04:00 0HLFR3ICVB1RH:00000001 [INF] The file "/scripts.e037fa6b4f7e71ff8752.js" was not modified (f1f8d725)
2018-08-05T17:44:21.4989785-04:00 0HLFR3ICVB1RG:00000001 [INF] Request finished in 338.7106ms 304 text/css (791a596a)
2018-08-05T17:44:21.6106213-04:00 0HLFR3ICVB1RI:00000001 [INF] Request finished in 364.1466ms 304 application/javascript (791a596a)
2018-08-05T17:44:21.6320851-04:00 0HLFR3ICVB1RH:00000001 [INF] Request finished in 359.3487ms 304 application/javascript (791a596a)
2018-08-05T17:44:21.6589312-04:00 0HLFR3ICVB1RD:00000002 [INF] Request finished in 469.0185ms 304 application/javascript (791a596a)
2018-08-05T17:44:21.7793177-04:00 0HLFR3ICVB1RE:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:44344/api/AccountApi/IsTokenValid   (ca22a1cb)
2018-08-05T17:44:21.8143086-04:00 0HLFR3ICVB1RG:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44344/   (ca22a1cb)
2018-08-05T17:44:21.8321622-04:00 0HLFR3ICVB1RH:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44344/assets/cv-image.png   (ca22a1cb)
2018-08-05T17:44:21.8589107-04:00 0HLFR3ICVB1RE:00000003 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.8710176-04:00 0HLFR3ICVB1RG:00000002 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.8757202-04:00 0HLFR3ICVB1RI:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44344/arrow.6d1d8006f736026b47b5.png   (ca22a1cb)
2018-08-05T17:44:21.9051720-04:00 0HLFR3ICVB1RD:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:44344/assets/folder.png   (ca22a1cb)
2018-08-05T17:44:21.9165040-04:00 0HLFR3ICVB1RH:00000002 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.9406308-04:00 0HLFR3ICVB1RG:00000002 [INF] The file "/index.html" was not modified (f1f8d725)
2018-08-05T17:44:21.9567901-04:00 0HLFR3ICVB1RI:00000002 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.9832620-04:00 0HLFR3ICVB1RD:00000003 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-08-05T17:44:21.9937909-04:00 0HLFR3ICVB1RH:00000002 [INF] The file "/assets/cv-image.png" was not modified (f1f8d725)
2018-08-05T17:44:22.0384755-04:00 0HLFR3ICVB1RI:00000002 [INF] The file "/arrow.6d1d8006f736026b47b5.png" was not modified (f1f8d725)
2018-08-05T17:44:22.0808732-04:00 0HLFR3ICVB1RD:00000003 [INF] The file "/assets/folder.png" was not modified (f1f8d725)
2018-08-05T17:44:22.0951809-04:00 0HLFR3ICVB1RH:00000002 [INF] Request finished in 290.5161ms 304 image/png (791a596a)
2018-08-05T17:44:22.1262656-04:00 0HLFR3ICVB1RG:00000002 [INF] Request finished in 295.8346ms 304 text/html (791a596a)
2018-08-05T17:44:22.2085541-04:00 0HLFR3ICVB1RI:00000002 [INF] Request finished in 386.0179ms 304 image/png (791a596a)
2018-08-05T17:44:22.2243526-04:00 0HLFR3ICVB1RD:00000003 [INF] Request finished in 427.7656ms 304 image/png (791a596a)
2018-08-05T17:44:22.4389177-04:00 0HLFR3ICVB1RE:00000003 [INF] Executing action method "WebCV_Fiches.Controllers.AccountApiController.IsTokenValid (WebCV_Fiches)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-08-05T17:44:22.4783881-04:00 0HLFR3ICVB1RE:00000003 [INF] Executing JsonResult, writing value "{ ok = True }". (7bb75e80)
2018-08-05T17:44:22.4898963-04:00 0HLFR3ICVB1RF:00000001 [INF] Executing action method "WebCV_Fiches.Controllers.AccountApiController.DoLogin (WebCV_Fiches)" with arguments (["", "WebCV_Fiches.Services.LoginService", "WebCV_Fiches.Extensions.SigningConfigurationsExtensions", "WebCV_Fiches.Extensions.TokenConfigurationExtentions"]) - ModelState is Invalid (ba7f4ac2)
2018-08-05T17:44:24.9151797-04:00 0HLFR3ICVB1RE:00000003 [INF] Executed action "WebCV_Fiches.Controllers.AccountApiController.IsTokenValid (WebCV_Fiches)" in 125.1826ms (afa2e885)
2018-08-05T17:44:24.9199189-04:00 0HLFR3ICVB1RF:00000001 [INF] Executed action "WebCV_Fiches.Controllers.AccountApiController.DoLogin (WebCV_Fiches)" in 2508.556ms (afa2e885)
2018-08-05T17:44:24.9544808-04:00 0HLFR3ICVB1RE:00000003 [INF] Request finished in 3177.9236ms 200 application/json; charset=utf-8 (791a596a)
2018-08-05T17:44:25.1025256-04:00 0HLFR3ICVB1RF:00000001 [ERR] An unhandled exception has occurred: Object reference not set to an instance of an object. (e9c253af)
System.NullReferenceException: Object reference not set to an instance of an object.
   at WebCV_Fiches.Controllers.AccountApiController.DoLogin(LoginModel userLogin, LoginService login, SigningConfigurationsExtensions signingConfigurations, TokenConfigurationExtentions tokenConfigurations) in C:\Workspace\LGCV\AppCV-Fiches\WebCV-Fiches\Controllers\AccountApiController.cs:line 75
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Session.SessionMiddleware.<Invoke>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Session.SessionMiddleware.<Invoke>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.<Invoke>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.<Invoke>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext()
2018-08-05T17:44:29.0000573-04:00 0HLFR3ICVB1RF:00000001 [ERR] Object reference not set to an instance of an object.   at WebCV_Fiches.Controllers.AccountApiController.DoLogin(LoginModel userLogin, LoginService login, SigningConfigurationsExtensions signingConfigurations, TokenConfigurationExtentions tokenConfigurations) in C:\Workspace\LGCV\AppCV-Fiches\WebCV-Fiches\Controllers\AccountApiController.cs:line 75
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Session.SessionMiddleware.<Invoke>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Session.SessionMiddleware.<Invoke>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.<Invoke>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.<Invoke>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext() (0f2dd115)
2018-08-05T17:44:50.7521556-04:00 0HLFR3ICVB1RF:00000001 [INF] Request finished in 30242.1141ms 500  (791a596a)
